src/main/webapp/ts/profileBuilder.ts

```diff
@@ -15,4 +22,22 @@ function setForm(value: string) {
             userTypeButton.setAttribute('value','Mentor');
         }
     }
-} 
\ No newline at end of file
+} 
+
+async function redirectifLoggedOut() {
```

Comment:
nit: function name should be camel case: redirectIfLoggedOut

Reply:
Renamed the function to fulfill the camel case